lacework/README.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Software composition analysis using Lacework
+description: Perform a Software Composition Analysis in your pipeline using Lacework.
+author: Buildkite
+use_cases: ["Security", "CI", "Schedule"]
+tools: ["Lacework"]
+primary_emojis: [:lacework:]
+plaforms: []
+languages: []
+---
+
+# Scan using Lacework
```

Comment:
Actually, I think we should switch this shorter tittle with the one above. The longer version would probably read better as the main page title and the more condensed version for the listing page.

Alternatively, we could make them the same.

Reply:
Yeah I like this - updated the README with those suggestions and then I'll merge.